release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2476,6 +2476,7 @@ openshift.io/scc: hostmount-anyuid
 
 * Due to the inclusion of old images in some image indexes, running `oc adm catalog mirror` and `oc image mirror` might result in the following error: `error: unable to retrieve source image`. As a temporary workaround, you can use the `--skip-missing` option to bypass the error and continue downloading the image index. For more information, see link:https://access.redhat.com/solutions/6975305[Service Mesh Operator mirroring failed].
 
+* It is not possible to create a macvlan on the physical function (PF) when a virtual function (VF) already exists. This issue affects the Intel E810 NIC. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2120585[BZ#2120585])
```

Comment:
Google results are showing a lot of "`macvlan`" (monospace) and some "MacVLAN". Do you know what the official styling of this should be? Also, is it a "`macvlan` network"? Just thinking it might have some noun that "`macvlan`" is describing.

Reply:
@jeana-redhat I did a search on docs before adding to see what it is, and it seems to be macvlan across the docs.  I think we are OK with macvlan as I think it well understood this is the CNI plugin original content already merged to 4.12 RN so wold like to keep this the same.